xbmc/addons/AddonDatabase.cpp

```diff
@@ -481,7 +481,7 @@ bool CAddonDatabase::GetAddon(const std::string& addonID, const AddonVersion& ve
     if (m_pDS->eof())
       return false;
 
-    return GetAddon(m_pDS->fv(0).get_asInt(), addon);
+    return GetAddon(m_pDS->fv("id").get_asInt(), addon);
```

Comment:
Does this need to be `addons.id` or is `id` ok?

I don't actually know, I'm not much for SQL ;)

Reply:
if the field is not aliased -> `id` is enough, this is what is returned by the db.
if you `SELECT addons.version, repo.addonID AS repoID FROM addons` for example you need to pull **repoID**

Reply:
Cool, thanks for explaining 😉